templates/views/privacy.md

```diff
@@ -1,62 +1,69 @@
 
-## Security
-We take all reasonable measures to protect your User Personal Information, by using industry standard security technologies. In the event of a data breach, we will notify you promptly.
+Le reste du traitement des informations que nous effectuons est nécessaire dans le cadre de nos intérêts légitimes. Par exemple, pour des raisons de sécurité, nous devons conserver des traces des informations matérielles uniques de votre machine afin d'identifier des utilisateurs frauduleux tentant de masquer leur identité.
 
-## Resolving complaints
+Si vous voulez demander la destruction de vos données, veuillez nous envoyer un courriel à admin@faforever.com
 
-If you have any concerns about how FAForever is handling your User Personal Information, please let us know immediately at [admin@faforever.com](mailto:admin@faforever.com).
+## Partage d'information
 
-## Changes to our Privacy Statement
+Nous ne partageons pas votre information avec aucun tiers pour aucune raison, cependant votre pseudonyme tel que fourni lors de la création de compte et modifiable sur le site internet est à considérer comme public, et est donc partagé avec d'autres utilisateurs de plusieurs manières lors de votre utilisation du service.
 
-Most changes to this document are likely to be minor, but should they occur, we will notify you and provide at least 30 days of time prior to the changes taking effect.
+Nous ne divulguons aucune information personnelle à moins d'y être légalement forcés.
+
+Nous n’hébergeons pas de contenu publicitaire sur FAForever.com, mais nous pourrions éventuellement intégrer du contenu de sites tiers, qui pourraient contenir des publicités. Aucune publicité apparaissant sur FAForever.com n'est sponsorisée, gérée ou créée par FAForever.
+
+Nous pourrions partager vos Informations utilisateur personnelles si nous sommes impliqués dans une fusion, vente ou acquisition. Si un tel changement de propriété arrivait, il se produirait selon les conditions décrites dans ce document.
+
+## Sécurité
+
+Nous prenons toutes les mesures nécéssaires pour protéger vos Informations Utilisateur Personnelles, par l'utilisation des standards industriels pour la sécurité des technologies. En cas de brèche de données, nous vous en informerons rapidement.
+
+## Gestion des plaintes
+
+Si vous avez des plaintes concernant la façon dont FAForever gère vos Informations Utilisateur Personnelles, veuillez nous le faire savoir immédiatement à admin@faforever.com
+
+## Modifications de notre déclaration de confidentialité
+
+La plupart des changements de ce document ont de bonnes chances d'être très mineurs, mais s'il devaient se produire, vous en serez informés et disposerez d'au moins 30 jours de délai avant l'application desdits changements.
```

Comment:
The privacy statement should end with a section clarifying that the English version is the "authority" should there be any dispute in wording/loss of translation.

Reply:
See the [german translation](https://github.com/FAForever/website/pull/143/files#diff-6f6adf454a28ca7deedb2bc3174e7189R73) for an example :)

Reply:
Okay, will do